frontend/lib/src/components/shared/Toolbar/Toolbar.tsx

```diff
@@ -40,17 +40,15 @@ export interface ToolbarActionProps {
   onClick: () => void
 }
 
-export function ToolbarAction({
-  label,
-  show_label,
-  icon,
-  onClick,
-}: ToolbarActionProps): ReactElement {
+export function ToolbarAction(props: ToolbarActionProps): ReactElement {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const { label, show_label, icon, onClick } = props
+
   const theme: EmotionTheme = useTheme()
 
   const displayLabel = show_label ? label : ""
   return (
-    <div data-testid="stElementToolbarButton">
+    <div data-testid={"stElementToolbarButton"}>
```

Comment:
nit: I think for strings you don't need the `{ }` brackets. Is this something a linter or so highlighted? In this case, I would vote for filing a tech debt item and do it consistently across the code base

Reply:
nah, i'll revert this. it's because it used to have a `props["data-testid"] ??` sitting in it